depthai_nodes/ml/messages/creators/regression.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+
+from ...messages import Prediction, Predictions
+
+
+def create_regression_message(predictions: List[float]) -> Predictions:
+    """Create a DepthAI message for prediction models.
+
+    @param prediction: Predicted value(s).
+    @type prediction: List[float]
```

Comment:
docstrings have `prediction` and the function parameter is `predictions`.

Reply:
Added in [db7505d](https://github.com/luxonis/depthai-nodes/pull/77/commits/db7505d9111b91c5a64e865f35638979ce3e2f43).